plugin/teksi_wastewater/teksi_wastewater_plugin.py

```diff
@@ -311,24 +311,21 @@ def initGui(self):
 
         self.network_layer_notifier.layersAdded([])
 
+        # check if database defaults are set
+        self.iface.projectRead.connect(self.tww_validity_check)
```

Comment:
This is actually done by plugin loading, but should be done on project opening otherwise we don't have the connection infos